src/main/cliapp/src/components/Autocomplete/AutocompleteEditor.js

```diff
@@ -26,10 +26,18 @@ export const AutocompleteEditor = (
 	const itemTemplate = (item) => {
 		if(valueDisplay) return valueDisplay(item, setAutocompleteHoverItem, op, query);
 
+		let nameValue = '';
+		if (item.geneFullName) {
+			nameValue = item.geneFullName.displayText;
+		} else if (item.alleleFullName) {
+			nameValue = item.alleleFullName.displayText;
+		} else if (item.name) {
+			nameValue = item.name;
+		}
```

Comment:
We attempted to move this kind of logic out of the Autocomplete thats the whole point of the valueDisplay method was to have this kind of logic. @adamgibs ?

Reply:
ah, I'd missed the valueDisplay stuff - I think I can undo these changes then

Reply:
logic removed